styles/audioPlayerStyle.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var React = require('react-native');
+
+var {
+  StyleSheet,
+} = React;
+
+module.exports = StyleSheet.create({
```

Comment:
Perché `module.exports`? Usiamo ES imports da altre parti, dovremmo cercare di essere consistenti nell'utilizzo per evitare confusione.

https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm

```suggestion
export default StyleSheet.create({
```

Reply:
Il problema è che con export esce un `TypeError: "exports" is read-only` sulla build web, ho paura che react native faccia un po' a cazzotti con l'idea degli stili globali (come avevi predetto 😱)